docs/walkthrough_train.md

```diff
@@ -3,15 +3,142 @@ title: Train models with eScriptorium
+- Once the form is filled, click on "train".
+
+The training task is now placed in a queue and will start soon. When training is finished, you will receive a notification.  
+
+!!! Warning
+    We do not recommend checking the "Overwrite the existing model file" since it will erase the original model.
+
+![image: starting a training task for a text recognizer in eScriptorium](img/train/train_recognizer.gif "Train a text recognizer in eScriptorium before cancelling the task.")
+
+## Train a segmentation model
+
+To train a segmentation model in eScriptorium, the process is the same as previously described. You need [training data](walkthrough_segment.md) to train a segmentation model. All the mobilized images/segment annotations must be in the same document.  
+
+From the "Images" tab:
+
+- Select the document parts you want to use to train the model,
+- Click on "Train", then select "Segmenter",
+- In the form, give an explicit name to your new model and/or select an existing model to perform [fine tuning](#fine-tuning),
+- Click on "Train"
+
+!!! Tip
+    Fine tuning [blla.mlmodel](https://github.com/mittagessen/kraken/blob/master/kraken/blla.mlmodel), Kraken's default segmentation model, works better than training a segmentation model from scratch.  
+
+!!! Warning
+    We do not recommend checking the "Overwrite the existing model file" since it will erase the original model.
+
+![image: starting a training task in eScriptorium to create a segmenter](img/train/train_segmenter.gif "Start training a segmenter in eScriptorium and cancel the task.")
+
+## Training parameters
+
+eScriptorium relies on Kraken's default parameters for training. Such parameters can be found browsing [Kraken's documentation](https://kraken.re/). However, it is important to note that **when [fine tuning](#fine-tuning) a model, Kraken inherits the base model's training parameters**.
+
+If you want to use more advanced training parameters, you should consider using Kraken in command line.  
+
+## Monitoring training
+
+Training can take several hours, you can normally monitor the evolution of a training task using two different approaches.
+
+### Using model listings  
+
+There are two ways to access a list of models:
+
+- in the "My Models" page (also accessible via `{base_url}/models`), all the models belonging to or shared with a user are listed.  
+- in a document's "Models" tab, only the models associated with the current document are listed.  
+
+In both of these display, you can see the "Training Status" of a model: either currently training or "done".  
+
+When a model is trained on eScriptorium, you can see the models resulting from each epoch by clicking on the "Toggle Versions" button. It will display a list of all the epochs along with their accuracy (and corresponding error count). It is then possible to download each model as an individual file.
```

Comment:
You should maybe do a reference to the definition of “epoch” in the vocabulary at the end

Reply:
Same with accuracy probably !

Reply:
Good catch, it was actually the initial reason why I created the vocabulary!